src/main/java/software/amazon/awssdk/sar/ClientBuilderUsingRetryStrategy.java

```diff
@@ -0,0 +1,154 @@
+package software.amazon.awssdk.sar;
+
+import software.amazon.awssdk.awscore.retry.AwsRetryStrategy;
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
+import software.amazon.awssdk.core.exception.SdkException;
+import software.amazon.awssdk.core.internal.retry.RetryPolicyAdapter;
+import software.amazon.awssdk.core.internal.retry.SdkDefaultRetryStrategy;
+import software.amazon.awssdk.core.retry.RetryMode;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.core.retry.RetryUtils;
+import software.amazon.awssdk.retries.AdaptiveRetryStrategy;
+import software.amazon.awssdk.retries.api.BackoffStrategy;
+import software.amazon.awssdk.retries.api.RetryStrategy;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+/**
+ * Building clients using {@link RetryStrategy} is similar to using a {@link RetryPolicy}.
+ */
+public class ClientBuilderUsingRetryStrategy {
+    /**
+     * Using retry mode is handy and supported by
+     * {@link ClientOverrideConfiguration.Builder#retryStrategy(RetryMode)}
+     */
+    public static DynamoDbClient clientWithRetryStrategy01() {
+        return DynamoDbClient
+                .builder()
+                .overrideConfiguration(o -> o.retryStrategy(RetryMode.STANDARD))
+                .build();
+    }
+
+    /**
+     * The new {@link RetryMode#ADAPTIVE} is <em>also</em> supported by retry strategies, but we don't have a concrete
```

Comment:
Do you mean the old one, not the new one?

Reply:
Yes, I meant to say *old*.